generators/server/templates/src/main/java/package/web/rest/_AccountResource.java

```diff
@@ -423,53 +440,58 @@ public void changePassword(@RequestBody String password) {
     * @param series the series of an existing session
     * @throws UnsupportedEncodingException if the series couldnt be URL decoded
     */
-   @DeleteMapping("/account/sessions/{series}")
-   @Timed
-   public void invalidateSession(@PathVariable String series) throws UnsupportedEncodingException {
-       String decodedSeries = URLDecoder.decode(series, "UTF-8");
-       userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin()).ifPresent(u ->
-           persistentTokenRepository.findByUser(u).stream()
-               .filter(persistentToken -> StringUtils.equals(persistentToken.getSeries(), decodedSeries))<% if (databaseType === 'sql' || databaseType === 'mongodb') { %>
-               .findAny().ifPresent(t -> persistentTokenRepository.delete(decodedSeries)));<% } else { %>
-               .findAny().ifPresent(persistentTokenRepository::delete));<% } %>
-   }<% } %>
-
-   /**
+    @DeleteMapping("/account/sessions/{series}")
+    @Timed
+    public void invalidateSession(@PathVariable String series) throws UnsupportedEncodingException {
+        String decodedSeries = URLDecoder.decode(series, "UTF-8");
+        userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin()).ifPresent(u ->
+            persistentTokenRepository.findByUser(u).stream()
+                .filter(persistentToken -> StringUtils.equals(persistentToken.getSeries(), decodedSeries))<% if (databaseType === 'sql' || databaseType === 'mongodb') { %>
+                .findAny().ifPresent(t -> persistentTokenRepository.delete(decodedSeries)));<% } else { %>
+                .findAny().ifPresent(persistentTokenRepository::delete));<% } %>
+    }<% } %>
+
+    /**
     * POST   /account/reset-password/init : Send an email to reset the password of the user
     *
     * @param mail the mail of the user
     * @throws EmailNotFoundException 400 (Bad Request) if the email address is not registered
     */
-   @PostMapping(path = "/account/reset-password/init")
-   @Timed
-   public void requestPasswordReset(@RequestBody String mail) {
-       mailService.sendPasswordResetMail(
-           userService.requestPasswordReset(mail)
-               .orElseThrow(EmailNotFoundException::new)
-       );
-   }
+    @PostMapping(path = "/account/reset-password/init")
```

Comment:
Why ? This code is fine according to Sonar !

Reply:
On our Sonar Docker image yes, but not on SonarCloud, which uses the "enterprise" version (with more checks): https://sonarcloud.io/project/issues?id=io.github.jhipster.sample%3Ajhipster-sample-application&resolved=false&types=BUG

Reply:
No but here, even on SonarCloud !!

Reply:
There are only 3 orElseThrow to change, not all of them !

Reply:
It's because the other are from other options, which are not selected on this SonarCloud config. All `.orElseThrow` have this issue, because they return a value we do not use. I know this rule is stupid, of course.

Reply:
I don't agree : we use the return valur of orElseThrow

Reply:
And Sonar doesn't complain